### ISSUE TITLE
[editor] Do we need separate "Add business" and "Add Place to OpenStreetMap"?

### ISSUE THREAD

**Issue by vng** [organicmaps/organicmaps#9798]
Some time ago we renamed _placepage_add_place_button_ to not confuse users (clear statement about editing OSM)

We still have a separate _placepage_add_business_button_. It is invoked when selecting a building (maybe ok) or selecting an area POI which is also a building (not ok). Do we really need to suggest "Add business" on area-building POI? Or remove it completely and keep only _placepage_add_place_button_ ?

CC @map-per @RedAuburn

**Comment by RedAuburn** [comment]
i don't think the distinction is useful, it makes sense to only keep `Add Place` to me 👍️

**Comment by map-per** [comment]
I agree, having two options here is just unnecessarily complicated (see: https://github.com/organicmaps/organicmaps/issues/9019#issuecomment-2478798600)

_This issue is a duplicate of  #9019_

**Comment by biodranik** [comment]
In maps me there were good statistics confirming the need for a separate Add Business button. It is very familiar to many non-OSM and US users and may simplify the flow of onboarding a new OSM editor.

If we can get statistics that prove the claim: "Add Business button is not used by new and existing users", then we can safely remove it. Otherwise, it's safer to keep it.

@vng what are these area POIs that are also buildings? Can they contain other POIs inside?

**Comment by vng** [comment]
This dualism is strange and is a bug. We show 2 different names for one action depending not on "non-OSM-familiar" or "US-users", but depending on "we selected a building".
Why were we so confident renaming the first option then? :)

**Comment by biodranik** [comment]
You did not answer my question )

Add business wording is more business-centric, and it's an obvious choice for newcomers when they select a building (e.g. when planning a trip or walking to the destination) and don't see a business there where they are going.

Add business on buildings also had a useful limitation of placing an object inside of the building no matter what.

In theory, we can use this button to show a different UI or hints, for focusing on business-related fields in the first place compared to more generic OSM fields.

Google shows the "Add your business" button together with the "Add a place" button.

Compare it with a more generic "Add place" (in a new, non-geeky user context). Which place? Is a restaurant or shop a place?

**Comment by vng** [comment]
What question? Open OSM and find any area POI object with a building ..

This logic is controversial to all your statements from here https://github.com/organicmaps/organicmaps/pull/6747

**Comment by muralito** [comment]
> Google shows the "Add your business" button together with the "Add a place" button.

Mmmm? The argument that "Google do it" also works for features requested in other issues? :-)

**Comment by pastk** [comment]
I agree having two buttons for the same action is confusing.

To keep it familiar for users accustomed to "add business" we can rename the button to e.g. "Add Place / Business to OpenStreetMap", but it should be only one button :)

Alternatively for buildings POIs we can show a single button "Add a Business to OpenStreeMap", but it would make sense when a new POI is limited to the building area only, IMHO.

**Comment by pastk** [comment]
Also adding e.g. a toilet or a drinking water POI inside a building isn't really "adding a business" :)

**Comment by pastk** [comment]
Related (it was removed/re-added several times already :) )
- https://github.com/organicmaps/organicmaps/pull/8158
- #4321 
- #9002

**Comment by map-per** [comment]
Using "Add Place / Business to OpenStreetMap" everywhere is the best solution IMO. (The button text will probably span two lines though)
